lightning/src/ln/functional_tests.rs

```diff
@@ -4668,27 +4668,15 @@ pub fn bolt2_open_channel_sending_node_checks_part2() {
 
 	let node_b_id = nodes[1].node.get_our_node_id();
 
-	// BOLT #2 spec: Sending node must set funding_satoshis to less than 2^24 satoshis
-	let channel_value_satoshis=2^24;
-	let push_msat=10001;
-	assert!(nodes[0].node.create_channel(node_b_id, channel_value_satoshis, push_msat, 42, None, None).is_err());
-
 	// BOLT #2 spec: Sending node must set push_msat to equal or less than 1000 * funding_satoshis
 	let channel_value_satoshis=10000;
 	// Test when push_msat is equal to 1000 * funding_satoshis.
 	let push_msat=1000*channel_value_satoshis+1;
 	assert!(nodes[0].node.create_channel(node_b_id, channel_value_satoshis, push_msat, 42, None, None).is_err());
 
-	// BOLT #2 spec: Sending node must set set channel_reserve_satoshis greater than or equal to dust_limit_satoshis
-	let channel_value_satoshis=10000;
-	let push_msat=10001;
-	assert!(nodes[0].node.create_channel(node_b_id, channel_value_satoshis, push_msat, 42, None, None).is_ok()); //Create a valid channel
-	let node0_to_1_send_open_channel = get_event_msg!(nodes[0], MessageSendEvent::SendOpenChannel, node_b_id);
-	assert!(node0_to_1_send_open_channel.channel_reserve_satoshis>=node0_to_1_send_open_channel.common_fields.dust_limit_satoshis);
+	nodes[0].node.create_channel(node_b_id, 100_000, 0, 42, None, None).unwrap();
```

Comment:
Explain in the commit message why it is dead code?

Reply:
I was after why that code isn't usefully testing anything interesting. For example, why isn't testing 'Sending node must set funding_satoshis to less than 2^24 satoshis' useful?

Reply:
FWIW, it looks like we have test coverage of this check elsewhere (I added a panic to the relevant `Channel` check).